spec/displaynames.html

```diff
@@ -126,12 +210,16 @@ <h1>Internal slots</h1>
       </p>
 
       <ul>
-        <li>[[LocaleData]].[[&lt;_locale_&gt;]] must have a [[types]] field for all locale values _locale_. The value of this field must be a Record, which must have fields with the names of all display name types: *"language"*, *"region"*, *"script"*, and *"currency"*.</li>
-        <li>The value of the fields *"language"*, *"region"*, *"script"*, and *"currency"* must be Records, which must have fields with the names of all display name styles: *"narrow"*, *"short"*, and *"long"*.</li>
+        <li>[[LocaleData]].[[&lt;_locale_&gt;]] must have a [[types]] field for all locale values _locale_. The value of this field must be a Record, which must have fields with the names of all display name types: *"language"*, *"region"*, *"script"*, *"currency"*, `"calendar"`, and `"dateTimeField"`.</li>
+        <li>The value of the field `"language"` must be a Record which must have fields with the names of one of the valid language displays: `"dialect"` and `"standard"`.</li>
+        <li>The language display fields under display name type `"language"` should contain Records which must have fields with the names of one of the valid display name styles: `"narrow"`, `"short"`, and `"long"`.</li>
+        <li>The value of the fields *"region"*, *"script"*, *"currency"*, `"calendar"`, and `"dateTimeField"` must be Records, which must have fields with the names of all display name styles: *"narrow"*, *"short"*, and *"long"*.</li>
         <li>The display name style fields under display name type *"language"* should contain Records with keys corresponding to language codes matching the `unicode_language_id` production. The value of these fields must be string values.</li>
```

Comment:
I think this line needs to be reworded a bit? Right now it seems conflicting with the line https://github.com/tc39/ecma402/pull/622/files#diff-e523319a519e9b301a04564d9f821c6e35720c21b603d77b5c0d982e1bfb5c53R215.

Reply:
sorry, I think you put in the wrong link. I cannot see what you try to say about the conflict